pkg/networkmanager/vlan.jsx

```diff
@@ -88,7 +88,8 @@ export const VlanDialog = ({ connection, dev, settings }) => {
         <NetworkModal dialogError={dialogError}
                       idPrefix={idPrefix}
                       onSubmit={onSubmit}
-                      title={_("VLAN settings")}
+                      title={!connection ? _("Add VLAN") : _("Edit VLAN settings")}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19209-20230829-062057-dc4a837a-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)